qutebrowser/mainwindow/mainwindow.py

```diff
@@ -94,13 +94,15 @@ def get_window(via_ipc, force_window=False, force_tab=False,
     return window.win_id
 
 
-def raise_window(window):
+def raise_window(window, alert=False):
```

Comment:
I think you meant to say `alert=True` here, to make all existing usages keep the old behavior :)

Reply:
whoops, you're right. Changed it